Tests/libhostmgrTests/resources/buildkite-environment-variables-basic-expected-output.txt

```diff
@@ -0,0 +1,19 @@
+source ~/.circ
```

Comment:
Do you think it's worth adding `#!/bin/bash` and run this script as an executable script, rather than running it using arbitrary shell assigned to the current user?

Reply:
This is an interesting question – it _might_ be safer in theory, because right now the code that runs the script is:

```
SCRIPT_PATH=$(hostmgr generate buildkite-job)
ssh builder@"$IP_ADDRESS" < "$SCRIPT_PATH"
```

Thus, it does indeed use the default shell, and that may change in future versions of macOS. 

I think this is still worth doing, but IMHO it'd make sense to do this as a separate change given that this is already significantly adjusting the logic for this script – if we accidentally break something, I'd like to keep it easy to narrow down!

Worth a new issue to track this though, because it could bite us eventually!

Reply:
Yeah, agree. Out of scope for this PR. I've created https://github.com/Automattic/hostmgr/issues/30 to track this.